### PR TITLE
Finish localization of SDK

### PR DESCRIPTION
**Pull request by nguerrera**
Build and deploy satellites from translations

The satellites are still not being picked up correctly by core MSBuild, but they work with desktop MSBuild. I am still investigating but I believe this is due to https://github.com/Microsoft/msbuild/issues/1476.

Change the target SatelliteDllsProjectOutputGroup to SatelliteDllsProjectOutputGroupWithFinalOutputPath, which will include the resources from the final output path rather than the intermediate path and hence will include the signed satellite assemblies

**Customer scenario**
Build messages from the SDK will not be localized without this change

**Bugs this fixes:**
SDK portion of [VSO 279745](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/279745)
(The project system portion was already completed before entering escrow)

**Workarounds, if any**
None

**Risk**
Low

**Performance impact**
None for English. Only the required small impact of loading satellite assembly for non-English languages.

**Is this a regression from a previous update?**
No. This was feature work approved to be done by RC3.

**How was this bug found?**
N/A. This was known work that wasn't done until now.

@srivatsn @dotnet/project-system

### CODE REVIEW

**Comment by nguerrera** [comment]
@dotnet-bot Test Windows_NT Release please

This appears to be #475

**Comment by nguerrera** [comment]
@natidea

**Review by srivatsn — APPROVED** [review]

**Comment by nguerrera** [comment]
@MattGertz for approval

File: build/Signing/SignToolConfig.json [hunk 1]
```diff
@@ -4,7 +4,20 @@
             "certificate": "Microsoft402",
             "strongName": "MsSharedLib72",
             "values": [
-                "Microsoft.NET.Build.Tasks.dll"
+                "Raw/Microsoft.NET.Build.Tasks.dll",
```
Comment (natidea) [inline build/Signing/SignToolConfig.json]: >Microsoft.NET.Build.Tasks.dll [](start = 21, length = 29)

So did the location of this DLL move, or is it in both places? Did you check that it is still packaged properly etc.

**Review by natidea — APPROVED** [review]

**Comment by natidea** [comment]
Approved to merge

File: build/Signing/SignToolConfig.json (hunk 1 above)
Comment (nguerrera) [inline build/Signing/SignToolConfig.json]: It moved.

**Pull request merged**